pkg/components/componenets_config.yaml

```diff
@@ -0,0 +1,25 @@
+componenets:
+  multus:
+    componentName: "multus"
+    Version: "0.2.0"
```

Comment:
Use lower case "v".

Also, some of the tags start with "v", not the number alone. Please check.